RAKwireless WisNode Documentation (#915)
helium/docs: docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx

---
id: rak7201
sidebar_label: RAK7201 WisNode Button 4K
title: RAK7201 WisNode Button 4K
description: The RAK7201 WisNode Button 4K is a remote wireless trigger device that supports user-defined functions for each key and is based on the LoRaWAN 1.0.2 protocol.
slug: /use-the-network/devices/ready-to-use/rakwireless/rak7201
---

import useBaseUrl from "@docusaurus/useBaseUrl";

<img
  src={useBaseUrl(
    "img/use-the-network/devices/ready-to-use/rakwireless/rak7201/RAK7201.png"
  )}
/>

## Overview

The RAK7201 WisNode Button 4K is a remote wireless trigger device. It supports user-defined functions for each key and is based on the LoRaWAN 1.0.2 protocol. Supported LoRaWAN bands are IN865, EU868, AU915, US915, KR920, and AS923.

> This WisNode Button is suitable for a variety of Smart Home applications: entertainment system control, control of lights, a snooze button for the alarms, or a remote trigger. With long-range wireless connectivity, amazing battery life, and four programmable buttons, there are a ton of applications this device can fit into.

Reviewer: Hi @kneelyrac I suggest we change this paragraph a bit as its a bit hard to understand. I like using the https://hemingwayapp.com/ to help me come up with more bold and clear sentences :)

I attached a screenshot of what the hemminwayapp was saying.

<img width="1512" alt="Screen Shot 2022-06-13 at 11 35 44 AM" src="https://user-images.githubusercontent.com/19807431/173391107-545558da-4398-4b54-9431-459132bb5740.png">

Below I think I was able to improve it a bit. Let me know what you think.

Author: Thank you for your suggestion, @Perronef5. :)

> This WisNode Button is suitable for a variety of Smart Home applications: entertainment system control, control of lights, a snooze button for the alarms, or a remote trigger. With long-range wireless connectivity, amazing battery life, and four programmable buttons, there are a ton of applications this device can fit into.

Reviewer: ```suggestion
This WisNode Button is suitable for a variety of Smart Home applications. Including, but not limited to, entertainment system control, control of lights, a snooze button for the alarms,  or a remote trigger. A device with long-range wireless connectivity, amazing battery life, and four programmable buttons. There are a tons of applications this device can fit into.
```

### Product Features

- Four (4) buttons with LED indicator
- Product size: 43 x 57 x 15 mm
- LoRaWAN® 1.0.2 compatible
- Battery: 350 mAh
- Battery working time: 1 year (Rechargeable)
- Power consumption: 
    - Transmit: 77.4-153 mA (Transmit current depends on factors such as transmit power, SF and Payload Length)
    - Standby: 4.8 uA
- Operating temperature: -20º C to 60º C
- Working environment: Indoor

### Useful Resources

- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Quickstart)
- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Datasheet/)
- [AT Command Manual](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/AT-Command-Manual)
- [Firmware Upgrade Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Firmware-Upgrade-Guide/)

> - [RAK7201 WisNode Button 4K](https://store.rakwireless.com/products/wisnode-button-4k-rak7201?utm_source=RAK7201WisNodeButton4K&utm_medium=Document&utm_campaign=BuyFromStore)

Reviewer 2: ```suggestion
- [RAK7201 WisNode Button 4K](https://store.rakwireless.com/products/wisnode-button-4k-rak7201)
```

cleaning up links

## Connecting to Helium (The People's Network)

> :::tip
> Before going through each and every step in connecting the RAK7201 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Quickstart/).
> :::

Reviewer 2: ```suggestion
:::tip

Before going through each and every step in connecting the RAK7201 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Quickstart/).

:::
```

padding white space for formatting
